recipes/entrypoints/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+cp entrypoints.py $SP_DIR
```

Comment:
> Not sure what `flonda` does though.

`flit` for `conda` :stuck_out_tongue_winking_eye: 

https://github.com/takluyver/flonda